test/e2e/clusterclass_changes_test.go

```diff
@@ -63,6 +63,29 @@ var _ = Describe("When testing ClusterClass changes [ClusterClass]", func() {
 					},
 				},
 			},
+			// ModifyMachinePoolBootstrapConfigTemplateFields are the fields which will be set on the
+			// BootstrapConfigTemplate of all MachinePoolClasses of the ClusterClass after the initial Cluster creation.
+			// The test verifies that these fields are rolled out to the MachinePools.
+			ModifyMachinePoolBootstrapConfigTemplateFields: map[string]interface{}{
+				"spec.template.spec.verbosity": int64(4),
+			},
+			// ModifyMachinePoolInfrastructureMachineTemplateFields are the fields which will be set on the
+			// InfrastructureMachinePoolTemplate of all MachinePoolClasses of the ClusterClass after the initial Cluster creation.
+			// The test verifies that these fields are rolled out to the MachinePools.
+			ModifyMachineInfrastructureMachinePoolTemplateFields: map[string]interface{}{
+				"spec.template.spec.extraMounts": []interface{}{
```

Comment:
the full-e2e job is failing with `failed to get field "spec.template.spec.extraMounts"`

Reply:
`spec.template.spec.template.extraMounts` should work